### PR TITLE
Add a few query tests

### LINKED ISSUE #63: Add test: Searching for queries with special characters in the name
### Prerequisites

- User is logged in
- Query with special characters in the name

### UI Test

Action | Result
------------ | -------------
Navigate to landing page | Page displays "Search queries..." input and button
Input the name of a query that contains special characters and click button | Browser opens queries page, "Search" tab is selected, page contains table, table contains only relevant items, the query that contains special characters in the name is displayed

### PR DESCRIPTION
**Pull request by jrbenny35**
Fixes #63 #62 #64 #61 #65 #59

### CODE REVIEW

File: tests/test_queries.py [hunk 1]
```diff
@@ -15,14 +16,93 @@
         ("Ashleys Query", "Query created by Ashley."),
     ],
 )
-def test_query(
+def test_query_by_username(
     create_queries,
```
Comment (hackebrot) [inline tests/test_queries.py]: Can you please update this to:

```text
create_queries: typing.Callable[..., None],
```

**Comment by jrbenny35** [comment]
I think asserting the number of results could work but what if they query is like 'default query' but there is one named 'default query 1' ?

**Comment by hackebrot** [comment]
That depends on the expected behavior for the search feature of Redash.

Thoughts @madalincm?

**Comment by jrbenny35** [comment]
Just trying it myself, they both show up.

**Pull request merged**